src/App/Pages/Accounts/SetPasswordPageViewModel.cs

```diff
@@ -86,10 +93,17 @@ public MasterPasswordPolicyOptions Policy
         public Action SetPasswordSuccessAction { get; set; }
         public Action CloseAction { get; set; }
         public string OrgIdentifier { get; set; }
+        public string OrgId { get; set; }
 
         public async Task InitAsync()
         {
             await CheckPasswordPolicy();
+            
+            var org = await _userService.GetOrganizationByIdentifierAsync(OrgIdentifier);
+            OrgId = org?.Id;
+            var policyList = await _policyService.GetAll(PolicyType.ResetPassword);
+            var policyResult = _policyService.GetResetPasswordPolicyOptions(policyList, OrgId);
```

Comment:
What happens if `OrgId` is `null`?

Reply:
The `policyResult` will return default values, which will resolve the `ResetPasswordAutoEnroll` property to `false`